prediction_market_agent/agents/microchain_agent/prompts.py

```diff
@@ -1,20 +1,54 @@
-from enum import Enum
-
 from pydantic import BaseModel
 
+
+class FunctionsConfig(BaseModel):
+    include_learning_functions: bool = False
+    include_trading_functions: bool = False
+    include_universal_functions: bool = False
+    include_agent_functions: bool = False
+    include_job_functions: bool = False
+    include_sending_functions: bool = False
+    include_twitter_functions: bool = False
+    include_messages_functions: bool = False
+    include_nft_functions: bool = False
+
+
+class SystemPromptConfig(BaseModel):
+    name: str
+    system_prompt: str
+    functions_config: FunctionsConfig
+
+
 NON_UPDATABLE_DIVIDOR = "You can use the following functions:"  # Agent can update only part of the prompt before this string.
-SYSTEM_PROMPT = f"""Act as an autonomous agent with a goal to learn, survive and evolve.
+REQUIRED_PROMPT_ENDING_FOR_MICROCHAIN = f"""
```

Comment:
Every prompt had this boilerplate stuff at the end. I extracted it out, so now it's not needed and it is auto-appended into it while building an agent.